README.md

```diff
@@ -614,9 +524,64 @@ There's at least a dozen of them available, but none of them support tooltips an
 - [Devtools/daily](https://www.devtoolsdaily.com/graphviz/)
 - [Sketchviz](https://sketchviz.com/)
 
+### Using multiple keys
+If you want to use existing multiple keys (instead of ephemeral ones) you can pass them as part of the network configuration. In that case it's recommended to **not** read them from TOML file. If you need to read them for the filesystem/os it's best if you use environment variables.
+Once you've read them in a safe manner you should programmatically add them to Seth's Config struct (which safe parts can be read from TOML file). You can either add them directly to `Network`, if it's already set up, or you can add them to `Networks` slice to the network you intend to use.
+
+For example you could start by reading the TOML configuration first:
+```go
+cfg, err := seth.ReadCfg()
+if err != nil {
+    log.Fatal(err)
+}
+```
+
+Then read the private keys in a safe manner. For example from a secure vault or environment variables:
+```go
+var privateKeys []string
+var err error
+privateKeys, err = some_utils.ReadPrivateKeysFromEnv()
+if err != nil {
+    log.Fatal(err)
+}
+```
+and then add them to the `Network` you plan to use. Let's assume it's called `Sepolia`:
+```go
+for i, network := range cfg.Networks {
+    if network.Name == "Sepolia" {
+        cfg.Networks[i].PrivateKeys = privateKeys
+    }
+}
+```
+
+Or if you aren't using `[[Networks]]` in your TOML config and have just a single `Network`:
+```go
+cfg.Network.PrivateKeys = privateKeys
+```
+
+Or... you can use the convenience function `AddPksToNetwork()` to have them added to both the `Network` and `Networks` slice:
+```go
+updatedCount := cfg.AddPksToNetwork(privateKeys, "Sepolia")
```

Comment:
Should there be a difference between this readme and another one in `wasp_examples`?

Reply:
not really